lib/sensu/server/sandbox.rb

```diff
@@ -2,16 +2,16 @@ module Sensu
   module Server
     module Sandbox
       # Evaluate a Ruby expression within the context of a simple
-      # "sandbox", a Proc in a module method. Use the Ruby `$SAFE`
-      # level of 4 when the version of Ruby is less than 2.1.0. A
-      # single value is provided to the "sandbox".
+      # "sandbox", a Proc in a module method. As of Ruby 2.3.0,
+      # `$SAFE` no longer supports levels > 1, so its use has been
+      # removed from this method. A single value is provided to the
+      # "sandbox".
       #
       # @param expression [String] to be evaluated.
       # @param value [Object] to provide the "sandbox" with.
       # @return [Object]
       def self.eval(expression, value=nil)
         result = Proc.new do
-          $SAFE = (RUBY_VERSION < "2.1.0" ? 4 : 3)
```

Comment:
Ruby 2.3.0 doesn't support $SAFE levels > 1, so let's just remove it. We will need to document eval filter attributes with a disclaimer, "don't do silly things".